lipnet/lipreading/videos.py

```diff
@@ -191,11 +192,30 @@ def get_frames_mouth(self, detector, predictor, frames):
             mouth_frames.append(mouth_crop_image)
         return mouth_frames
 
-    def get_video_frames(self, path):
+    def get_video_frames_skvideo(self, path):
         videogen = skvideo.io.vreader(path)
+        #print("aaaa")
         frames = np.array([frame for frame in videogen])
         return frames
 
+
+    def get_video_frames(self, path):
```

Comment:
Rename the method to signify the use of opencv

Reply:
Done!